slamon_afm/app.py

```diff
@@ -1,21 +1,46 @@
 import logging
+
+import os
+
 from flask import Flask
 
 from slamon_afm.models import db
 from slamon_afm.routes import agent_routes, bpms_routes, status_routes, dashboard_routes
 
 
+def _log_level_from_str(log_level):
+    try:
+        return int(log_level)
+    except ValueError:
+        numeric_level = getattr(logging, log_level.upper(), None)
+        if not isinstance(numeric_level, int):
+            raise ValueError('Invalid log level: %s' % log_level)
+        return numeric_level
+
+
+def _bool_from_str(value):
+    if value.lower() in ['true', 'yes']:
```

Comment:
Is it necessary to raise a `ValueError`? I personally think this whole function is quite redundant since you can do just
```python
return value.lower() in ['true', 'yes']
```

This *does* have the side-effect that *everything* else (including 'false' and 'no') will return `False`.